matchmaking/backend/database/migrations.js

```diff
@@ -40,13 +85,16 @@ const runMigrations = async () => {
  * @param {number} migration - Which migration to target.
  */
 const readMigrationFiles = async (migration) => {
-  const baseDirectory = './database/migrations/';
+  const baseDirectory = "./database/migrations/";
   let data = [];
-  
+
   // Read migrations directory.
   fs.readdir(baseDirectory, (error, files) => {
     if (error) {
-      console.error(`${colors.Yellow}DB Migrations: ${colors.Pink}Error reading migrations directory${colors.Reset}`, error);
+      console.error(
+        `${colors.Yellow}DB Migrations: ${colors.Pink}Error reading migrations directory${colors.Reset}`,
+        error
```

Comment:
I think this could be refactored to a helper func, i.e.: `errLogOutputFormatter(areaOfConcern, errMsg)`

Or some such thing.

Reply:
Definitely at it wouldn't be hard. I'll add it to the tickets but not high priority.